### ISSUE TITLE
Res-Life cannot be cycled through with tab

### ISSUE THREAD

**Issue by Jshmill** [gordon-cs/gordon-360-ui#2727]
When you press tab, you should be able to cycle through most clickable content on a webpage, which may be helpful for those who are visually impaired and have text-to-speech enabled. 

https://github.com/user-attachments/assets/5aa4d89f-20dc-4963-b37c-07bef95e6ea2

As you can see in this video, the tab skips over Res-Life, which would make it difficult for said users to use that feature.
